hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeAdminMonitorImpl.java

```diff
@@ -297,6 +302,14 @@ private boolean checkContainersReplicatedOnNode(DatanodeDetails dn)
     LOG.info("{} has {} sufficientlyReplicated, {} underReplicated and {} " +
         "unhealthy containers",
         dn, sufficientlyReplicated, underReplicated, unhealthy);
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("{} has {} underReplicated [{}] and {} unhealthy [{}] " +
+              "containers", dn, underReplicated,
+          underReplicatedIDs.stream().map(
```

Comment:
You can do the join more simply with:

```
String.join(",", list);
```

Reply:
Actually, this might only work if the list is of type String. I am not 100% sure.

Reply:
I tried the String.join.  intellij raised the compile error.